index.rst

```diff
@@ -48,10 +48,205 @@
+source such as the all-sky camera has been stored in the Large File
+Annex, its avaibility is broadcast on the Large File Annex
+Announcement Even Stream.
+
+By volume, most of the information in the LFA is of no interest to Science Platform users, nor is it in a form that is tractable for python-level exploitation. For example, the LFA contains reports in the form of Excel spreadsheets; a Science Platform user is likely to create reports from the data directly, rather than interact with the derived documents.
+
+Data of interest in the LFA originates from:
+
+* The All-Sky Camera
+
+* Guider images
+
+* Composite Wavefrong Images
+
+* Laser (KSK: I'm unclear exactly what these laser data are.  Robert specifically said he didn't necessarily need the positioning laser data)
+
+* Flatfield screen monochrometer
+
+* Sky-spectrum monitor (if/when built)
+
+Like the data from the Auxillary Telescope, users want to interact with the LFA data through the butler, an appropriate dataset type having been define. Moreover users require these data with very low latencies as it is likely that they need it in order to make on-the-fly adjustments to systems during commissioning.
+
+We therefore propose that the LFA Announcement Stream is monitored by the DM-CS and when data from these enumerated sources is made available, that it be injected into the data backbone, from whence it will be treated like data (and not telemetry) by upstream services.
+
+
+Design-neutral Requirements
+===========================
+
+Rehardless of whether the ETL or new proposed architecture is adopted, the eventual architecture needs to show how it can meet satisfy the following requirments and use cases.  
+
+
+Availability of the DM-EFD capabilities
+----------------------------------------
+
+If, as anticipated, DM tooling is the primary of interface to EFD data for anyone beyond hardware-level engineers, availability of those services will be important to operational staff in Chile and the US, as well as to science users. It is therefore a requirement that the entire architecture is structured so that sandbox deployments, rolling upgrades and carefully coordinated downtime are the norm for routine operations. 
+
+Interfaces
+----------
+
+Data should be available via TAP/ADQL services as other data sources available to the Science Platform.
+
+The interface to the Science Platform should be deployment- and time-invariant: the same notebook accessing EFD data should run without modifications on the day in Chile and a month later at the LDF.
+
+A syntactic sugar to make access to EFD data more pythonic from the notebook (and to shield the user from schema implementation details) has been requested. Here is an example of how a notebook user could obtain statistics on the M1/M3 temperature sensors::
+
+  import numpy as np
+  import lsst.efd as efd
+  ...
+
+  # Get the temperatures in one go 
+  envtemp = efd.get("m1m3.actuators.envtemp")
```

Comment:
Isn't a date range needed here, at a minimum?  Or is this meant to be over all time?

Reply:
If a range is left off, I would assume it's all of them.